fedmsg/tests/test_commands.py

```diff
@@ -1,4 +1,6 @@
 import resource
```

Comment:
Not related to this PR, but it might be good to give this file a Copyright block.